docs/content/drafts/PageLayout.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: PageLayout
+status: Draft
+description: TODO
+---
+
+<Note variant="warning">Not implemented yet</Note>
+
+## Examples
+
+### Default
+
+```jsx
+<PageLayout>
+  <PageLayout.Header>Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane>Pane</PageLayout.Pane>
+  <PageLayout.Footer>Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+### With dividers
+
+```jsx
+<PageLayout>
+  <PageLayout.Header divider="line">Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane divider="line">Pane</PageLayout.Pane>
+  <PageLayout.Footer divider="line">Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+### Condensed
+
+```jsx
+<PageLayout columnGap="condensed" rowGap="condensed">
+  <PageLayout.Header>Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane>Pane</PageLayout.Pane>
+  <PageLayout.Footer>Footer</PageLayout.Footer>
+</PageLayout>
```

Comment:
I'm so sorry to throw a rabbit hole your way but wouldn't it be nice to use iframes in the examples so that we can add a [resize draggy thing](https://tailwindcss.com/#mobile-first) for folks to play with and see the effect.

Reply:
I would love to add something like this to Docotcat ❤️